Source/Scene/Batched3DModel3DTileContent.js

```diff
@@ -362,7 +362,8 @@ define([
             pickVertexShaderLoaded : getPickVertexShaderCallback(content),
             pickFragmentShaderLoaded : batchTable.getPickFragmentShaderCallback(),
             pickUniformMapLoaded : batchTable.getPickUniformMapCallback(),
-            addBatchIdToGeneratedShaders : (batchLength > 0) // If the batch table has values in it, generated shaders will need a batchId attribute
+            addBatchIdToGeneratedShaders : (batchLength > 0), // If the batch table has values in it, generated shaders will need a batchId attribute
+            pickInfo : {tile : content._tile}
```

Comment:
Remove all occurrences of `pickPrimitive` now that there's `pickInfo`. Also `pickObject` may be a better name now.

Also include `primitive` (= tileset) and `content` into the pick object. Create the pick object as a variable above so this area is cleaner.